scripts/build_example

```diff
@@ -41,28 +41,46 @@ else
     TARGET="$NAME"
 fi
 
+bazel_build_flags+=(
+    '--symlink_prefix=bazel-client-'
+    "--compilation_mode=$COMPILATION_MODE"
+)
+
 readonly RUST_FOLDER="$(find examples/"$TARGET" -type d -name rust)"
 if [[ -n "$RUST_FOLDER" ]]; then
     for module in examples/"$TARGET"/module*/rust/Cargo.toml; do
-    cargo build --release --target=wasm32-unknown-unknown --manifest-path="$module"
+        cargo build --release --target=wasm32-unknown-unknown --manifest-path="$module"
     done
-fi
 
-bazel_build_flags+=(
-    '--symlink_prefix=bazel-client-'
-    "--compilation_mode=$COMPILATION_MODE"
-)
+    # Serialize application configuration for Rust module.
+    if [[ "hello_world" != "$TARGET" ]]; then
+        bazel --output_base="$CACHE_DIR/client" build "${bazel_build_flags[@]}" "//examples/$TARGET/config:config"
+    else
+        # Build translator.
```

Comment:
Maybe explain a bit more why this is needed? Does not need to describe the entire architecture in detail, but at least justify why the build looks different in this case.

Reply:
Done